vite.config.ts

```diff
@@ -59,37 +59,39 @@ export const sharedConfig: UserConfig = {
   },
 }
 
-export default defineConfig(({ command }) => ({
-  ...sharedConfig,
-  base: command === 'serve' ? `http://localhost:${port}/` : '/dist/',
-  server: {
-    port,
-    hmr: {
-      host: 'localhost',
-    },
-  },
-  build: {
-    outDir: r('extension/dist'),
-    emptyOutDir: false,
-    sourcemap: isDev ? 'inline' : false,
-    // https://developer.chrome.com/docs/webstore/program_policies/#:~:text=Code%20Readability%20Requirements
-    terserOptions: {
-      mangle: false,
+export default defineConfig(({ command, mode }) => {
+  Object.assign(process.env, loadEnv(mode, process.cwd()))
+  return {
+    ...sharedConfig,
+    base: command === 'serve' ? `http://localhost:${port}/` : '/dist/',
+    server: {
+      port,
+      hmr: {
+        host: 'localhost',
```

Comment:
@giteshk01 this localhost here is fine?

Reply:
yes @vinit-tribes it's used when we are developing our app, it won't run when we bundle everything.